torch/nn/modules/conv.py

```diff
@@ -18,7 +17,7 @@ class _ConvNd(Module):
     def __init__(self, in_channels, out_channels, kernel_size, stride,
                  padding, dilation, transposed, output_padding,
                  groups, bias, padding_mode):
-        super(_ConvNd, self).__init__()
+        super(_ConvNdBase, self).__init__()
```

Comment:
What is the need to create _ConvNdBase? It seems to be identical to _ConvNd but for reset_parameters()? If you inherit from Conv2d, this change will not be needed.

Reply:
see comments above